src/checkout-buttons/checkout-button-options.ts

```diff
@@ -24,4 +24,9 @@ export interface CheckoutButtonInitializeOptions extends CheckoutButtonOptions {
      * omitted unless you need to support Braintree Credit.
      */
     braintreepaypalcredit?: BraintreePaypalButtonInitializeOptions;
+
+    /**
+     * The ID of a container which the checkout button should be inserted.
+     */
+    containerId: string;
```

Comment:
Not 100% sure, but I would expect any checkout button to have a `containerId` ?

Reply:
Also, modified this to be `containerId` instead of `container` so it's consistent with embeddable checkout. However I've noticed payment strategies use `container` instead of `containerId`. Thoughts?

Reply:
Yeah I guess it does make sense to have it as a common property. `CheckoutButtonInitializer` is still considered to be "alpha" so I guess we _could_ make a breaking change if there's a good reason. We just have to make a corresponding update in the client when we release the change.